main.py

```diff
@@ -418,7 +418,7 @@ def to_one_line(original):
 
     t = ast.parse(original)
 
-    original = original.strip()
+    ##original = original.strip()
```

Comment:
I think that if this is omitted, then a program equal to 

"""
print 'this program has unnecessary newlines'

"""

won't be properly detected by the code below in order to be lazily kept as the same single line.